qbft/spectest/tests/messages/marshal_justification_without_full_data.go

```diff
@@ -0,0 +1,50 @@
+package messages
+
+import (
+	"encoding/hex"
+
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// MarshalJustificationsWithoutFullData tests marshalling justifications without full data.
+func MarshalJustificationsWithoutFullData() tests.SpecTest {
+	ks := testingutils.Testing4SharesSet()
+
+	rcMsgs := []*qbft.SignedMessage{
+		testingutils.TestingRoundChangeMessageWithRoundAndFullData(ks.Shares[1], 1, 2, nil),
+	}
+
+	prepareMsgs := []*qbft.SignedMessage{
+		testingutils.TestingPrepareMessageWithRoundAndFullData(ks.Shares[1], types.OperatorID(1), 1, nil),
+	}
+
+	msg := testingutils.TestingProposalMessageWithParams(
```

Comment:
this has full data, the test shouldn't have full data

Reply:
I see that `nil` was passed for full data. So I think it is 🆗